bootstrap/011_warehouse_schedules.sql

```diff
@@ -94,3 +107,12 @@ EXCEPTION
         INSERT INTO internal.task_warehouse_schedule SELECT :this_run, FALSE, :task_outcome;
         RAISE;
 END;
+
+-- owners rights procedures can't get the timezone from the session parameters.
+CREATE OR REPLACE FUNCTION INTERNAL.GET_CURRENT_TIMEZONE()
+RETURNS STRING
+LANGUAGE JAVASCRIPT
+AS
+$$
+    return Intl.DateTimeFormat().resolvedOptions().timeZone;
```

Comment:
what does this return?

Reply:
This returns a timezone identifier like "America/New_York" (or similar).